core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiService.scala

```diff
@@ -144,9 +145,11 @@ trait EntityApiService extends UserInfoDirectives {
         } ~
         path("workspaces" / Segment / Segment / "entities" / Segment) { (workspaceNamespace, workspaceName, entityType) =>
           get {
-            traceRequest { span =>
+            // if any other APIs adopt streaming, move this implicit val higher up in the EntityApiService trait
+            implicit val jsonStreamingSupport: JsonEntityStreamingSupport = EntityStreamingSupport.json()
+            traceRequest { _ =>
               complete {
-                entityServiceConstructor(userInfo).listEntities(WorkspaceName(workspaceNamespace, workspaceName), entityType, span)
+                entityServiceConstructor(userInfo).listEntities(WorkspaceName(workspaceNamespace, workspaceName), entityType)
```

Comment:
opencensus tracing doesn't support tracing inside streams, so for this API we only have the outer `traceRequest`; no need to pass the span down into `listEntities`